docs/fundamentals/code-analysis/quality-rules/ca1001.md

```diff
@@ -41,6 +43,19 @@ To fix a violation of this rule, implement <xref:System.IDisposable> and from th
 
 In general, do not suppress a warning from this rule. It is OK to suppress the warning when the dispose ownership of the field(s) is not held by the containing type.
 
+## Configure code to analyze
```

Comment:
I have changed this title because API is weird when the configuration is about a modifier or parameter. WDYT?

Reply:
Seems fine to me, thanks.